urbanairship-react-native/android/build.gradle

```diff
@@ -6,7 +6,7 @@ def safeExtGet (prop, fallback) {
 }
 
 dependencies {
-    def airshipVersion = "13.1.2"
+    def airshipVersion = "13.2.0"
```

Comment:
Does this still need to be here now that it is in `common-build.gradle`?

Reply:
Indeed, this is not needed anymore, thanks 😉